docs/GettingStarted.md

```diff
@@ -64,115 +64,242 @@ block { display: none; }
 
 <!-- ######### MAC for iOS ##################### -->
 
-<block class="mac ios" />
+<block class="mac ios android" />
+
+## Installing Dependencies
 
-## Dependencies for Mac + iOS
+<block class="mac ios" />
 
-You will need Xcode, node.js, the React Native command line tools, and Watchman.
+You will need Node.js, Watchman, the React Native command line interface, and Xcode.
 
 <block class="mac android" />
 
-## Dependencies for Mac + Android
-
-You will need Android Studio, node.js, the React Native command line tools, and Watchman.
+You will need Node.js, Watchman, the React Native command line interface, and Android Studio.
 
 <block class="mac ios android" />
 
-We recommend installing node and watchman via [Homebrew](http://brew.sh/).
+### Node, Watchman
+
+We recommend installing Node and Watchman using [Homebrew](http://brew.sh/). Run the following commands in a Terminal after installing Homebrew:
 
 ```
 brew install node
 brew install watchman
 ```
 
-Node comes with npm, which lets you install the React Native command line interface.
+> [Watchman](https://facebook.github.io/watchman) is a tool by Facebook for watching
+changes in the filesystem. It is highly recommended you install it for better performance.
+
+### The React Native CLI
+
+Node.js comes with npm, which lets you install the React Native command line interface. Run the following command in a Terminal:
 
 ```
 npm install -g react-native-cli
 ```
 
-If you get a permission error, try with sudo: `sudo npm install -g react-native-cli`.
+> If you get a *permission error*, try using sudo: `sudo npm install -g react-native-cli`.
 
-If you get error `Cannot find module 'npmlog'`, try this before: `curl -0 -L http://npmjs.org/install.sh | sudo sh`.
+> If you get an error like `Cannot find module 'npmlog'`, try installing npm directly: `curl -0 -L http://npmjs.org/install.sh | sudo sh`.
 
 <block class="mac ios" />
 
-The easiest way to install Xcode is via the [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835?mt=12).
+### Xcode
+
+The easiest way to install Xcode is via the [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835?mt=12). Installing Xcode will also install the iOS Simulator and all the necessary tools to build your iOS app.
 
 <block class="mac android" />
 
+### Android Development Environment
+
+Setting up your development environment can be somewhat tedious if you're new to Android development. If you're already familiar with Android development, there are a few things you may need to configure. In either case, please make sure to carefully follow the next few steps.
+
+#### 1. Install Android Studio
+
 Download and install [Android Studio](https://developer.android.com/studio/install.html).
 
-If you plan to make changes in Java code, we recommend [Gradle Daemon](https://docs.gradle.org/2.9/userguide/gradle_daemon.html) which speeds up the build.
```

Comment:
I don't think this belongs in the Getting Started guide. After all, the goal is to get you up and running with a simple React Native app.

If you do intend to make changes to native Java code, then you'd likely read the Native Android Modules doc, which [now has these instructions](https://github.com/facebook/react-native/pull/9867/files/a0aab8b8671e8e76652ec55afd538b3e2d92d2ab#diff-bdf570846f463516068b23131b72eaafR15).